unit-tests-jest/ts/numbers.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers: number[]): number[] {
+  return numbers.filter((n) => (n + 1) % 2);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ */
+export function toDollars(amount: number): string {
+  return `${amount < 0 ? '-' : ''}$${Math.abs(amount).toFixed(2)}`;
+}
+
+/**
+ * Returns a new array of numbers where every entry has been divided by the
+ * given divisor. Does not modify the original array.
+ */
+export function divideBy(numbers: number[], divisor: number): number[] {
+  const newNumbers: number[] = [];
+  for (let i = 0; i < numbers.length; i++) {
+    newNumbers.push(Math.round((numbers[i] / divisor) * 100000) / 100000);
```

Comment:
I'm curious why you use the `* 100000 / 100000` trick? Why not just trust JS to do the right thing?

Reply:
In my test, I have a division of 9.9 / 3, which should be 3.3. However I was getting back something like 3.300000003, so I had to round it to match.